code/datums/gamemodes/conspiracy.dm

```diff
@@ -99,6 +103,18 @@
 	else
 		return candidates
 
+/datum/game_mode/conspiracy/proc/random_radio_frequency()
+	var/f = 0
+	var/list/blacklisted = list(0, 1451, 1457)
+	blacklisted.Add(R_FREQ_BLACKLIST_HEADSET)
+	blacklisted.Add(R_FREQ_BLACKLIST_INTERCOM)
```

Comment:
why not just put those all in the list??

Reply:
Because I copy pasted it!